pkg/metrics/pool.go

```diff
@@ -0,0 +1,60 @@
+package metrics
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+const (
+	poolGetCounterMetricsName = "pool_get_total"
+	poolPutCounterMetricsName = "pool_put_total"
+	poolSizeMetricsName       = "pool_put_size"
+)
+
+var (
+	poolGetCounter = prometheus.NewCounterVec(prometheus.CounterOpts{
```

Comment:
We should add UT for metrics exclude histogram.

Reply:
Added